### PR TITLE
{geo}[foss/2022a] CESM-deps v2

### PR DESCRIPTION
**Pull request by lexming**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by lexming** [comment]
@boegelbot please test @ jsc-zen2

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 7 out of 7 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/verdurin/cf4e01839fa061816f1f5023e8ffed40 for a full test report.

File: easybuild/easyconfigs/c/CESM-deps/CESM-deps-2-foss-2022a.eb [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+components = [
+    # install extra configuration tools and files for VSC clusters
+    ('cesm-config', '1.6.2', {
+        'easyblock': 'Tarball',
+        'source_urls': ['https://github.com/vub-hpc/%(name)s/archive'],
+        'sources': [{'download_filename': 'v%(version)s.tar.gz', 'filename': SOURCE_TAR_GZ}],
+        'checksums': ['eda865c6c6885253728de0d570a850a728310785162c8b78bd896b0d1dfbbf48'],
+        'start_dir': '%(name)s-%(version)s',
+    }),
+]
```
Comment (verdurin) [inline easybuild/easyconfigs/c/CESM-deps/CESM-deps-2-foss-2022a.eb:39]: @lexming is this part suitable for the upstream repo, or should it go in a hook for your site?

Comment (lexming) [reply easybuild/easyconfigs/c/CESM-deps/CESM-deps-2-foss-2022a.eb:39]: It does not hurt to install it, it just contains some configuration files and a script to update configuration files. It was already part of `CESM-deps-2-foss-2021b.eb`

**Review by smoors — APPROVED** [review]
lgtm

**Comment by smoors** [comment]
Going in, thanks @lexming!

**Pull request merged**